tests/test_config_command.rs

```diff
@@ -0,0 +1,58 @@
+// Copyright 2022 The Jujutsu Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use regex::Regex;
+
+use crate::common::TestEnvironment;
+
+pub mod common;
+
+#[test]
+fn test_config_list_single() {
+    // TODO: Populate expected config values explicitly in this file.
```

Comment:
You can do that using `test_env.add_config()`

Reply:
That won't let me remove keys though, will it? I was getting a long list that didn't look fully deterministic, so I was hoping to trim it way down before asserting snapshot on the entire output.

Reply:
Correct, but it will still help with testing by letting you add test-specific then check that they gets rendered exactly as you expect. You can add things like this:

```
[test-table]
key1 = 1
key2 = "string"
```

Reply:
Done. I tweaked env var precedence in `TestEnvironment` so I could do that without having to fork `jj_cmd_success` or have the values clobbered.